src/screens/WalletScreen/index.js

```diff
@@ -26,7 +28,11 @@ type Props = {
    * @desc React Native Navigation navigator object.
    */
   navigator: Navigator,
-}
+  /**
+   * @desc Current user object.
+   */
+  user: any,
```

Comment:
Please don't use `any`. It's almost always means that you need to find if there is an existing type for this or create a new type. Any kills the profit of using flow. You can use [`Account` type](https://github.com/Bit-Nation/BITNATION-Pangea-mobile/blob/develop/src/types/Account.js#L5) like that: `user: Account | null`